sml-bin/redeploy-testlib.sml

```diff
@@ -0,0 +1,50 @@
+structure Path =  OS.Path
+structure File =  OS.FileSys
+
+fun printLn x = print (x ^ "\n")
+
+fun fileExists x = File.access (x,[])
+
+fun testLibName base slug =
+  Path.joinDirFile
+    {
+      dir = (Path.concat (base,slug)),
+      file = "testlib.sml"
+    }
+
+fun dirList dirname =
+  let
+    fun go d xs =
+      case File.readDir d of
+          SOME x => go d (x::xs)
+        | NONE   => (File.closeDir d; xs)
+  in
+    go (File.openDir dirname) []
+  end
+
+fun fileBytes path =
+  let
+    val in' = BinIO.openIn path
```

Comment:
I considered changing it in case it is confusing, but using `'` as a `prime` like identifier modification is pretty common in the ML family.  For the record, I think the fact of general tooling being incapable of recognizing/capturing the expressiveness of a language is a poor reason to limit one's expressiveness when using the language.  If I rename it I am likely to name it `ins` which was a close second in my consideration.  I suppose `instream` isn't too long though.  I don't know why I felt so averse to a longer name.

Reply:
The longer name of `instream` would be preferable to any of the shorter names.  You could still use the `'` to indicate the additional information stated @guygastineau .

Reply:
Thanks, @kotp.  Yes, you are right about `instream`. I have pushed it now.  `'` didn't have any meaning in my usage which is why it was wrong.  I was using a more or less semantic naming convention as a hack to get around the inconvenience of a keyword.